src/validators/boolean.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable functional/no-this-expression */
+import { ValidationError } from '../errors';
+import { initialModifiers } from '../schema';
+import { typeToPrint } from '../stringify';
+import type { Schema } from '../types';
+
+export const boolean = () => {
+  return {
+    __modifiers: initialModifiers,
+    toString() {
+      return typeToPrint('boolean');
```

Comment:
Po co to? Ta funkcja zwraca po prostu `'boolean'` o ile się dobrze orientuję

Reply:
Abstraction